usr/local/share/bastille/bootstrap.sh

```diff
@@ -420,10 +420,10 @@ current-build-latest|current-BUILD-LATEST|CURRENT-BUILD-LATEST)
     PLATFORM_OS="HardenedBSD"
     validate_release_url
     ;;
-http?://github.com/*/*|http?://gitlab.com/*/*)
+http://*/*/*|https://*/*/*)
```

Comment:
`http?://*/*/*` would simplify this regex?

Reply:
Great point, I did try to simplify it with a `?` exactly like you have there but I couldn't get any URLs to match, it just printed the usage.

Reply:
I quickly setup Gitea to test this locally.
Working for me. Can you confirm that `http?://*/*/*)` for line 423 works?

Reply:
I'm still unable to get the shortened regex to work for a local Gitea testing instance. I changed line 423 to this: 

`http?://*/*/*)`

It just prints the usage message:

```
$ doas bastille bootstrap http://192.168.1.50:3000/macon-bastille/www
Usage: bastille bootstrap [release|template] [update|arch]
```

Reply:
Something to note: when I change the URL to `https://192.168.1.50:3000/macon-bastille/www`, with an `https` protocol, it works. It must just be that the `?` is requiring an extra character, like the `s`? But that seems like irregular behavior for `?`.

```
$ doas bastille bootstrap https://192.168.1.50:3000/macon-bastille/www                                                                                                                                                                               
Already up to date.
Detected Bastillefile hook.
[Bastillefile]:
PKG lighttpd
OVERLAY usr
SYSRC lighttpd_enable=YES
SERVICE lighttpd start
Template ready to use.
```